docs/products/flink/howto/connect-bigquery.rst

```diff
@@ -205,3 +205,26 @@ Following is an  example of a Google BigQuery SINK table:
 If the integration is successfully created, the service credentials and project id will be automatically populated in the Sink (if you have left them back as shown in the example above).
 
 
+Configure integration using Aiven Console
+--------------------------------------------
+
+If you're using Google BigQuery for your data storage and analysis, you can seamlessly integrate it as a sink for Aiven for Apache Flink streams. To achieve this via the `Aiven Console <https://console.aiven.io/>`_, follow these steps:
+
+1. Log in to `Aiven Console <https://console.aiven.io/>`_ and choose your project. 
+2. From the **Services** page, you can either :doc:`create a new Aiven for Apache Flink </docs/platform/howto/create_new_service>` service or select an existing service.
+3. Next, configure Google BigQuery service integration endpoint:
+  
+   * Navigate to the **Projects** screen where all the services are listed. 
+   * From the left sidebar, select **Integration endpoints**. 
+   * Select **Google Cloud BigQuery** from the list, and then select **Add new endpoint** or **Create new**. 
+   * Enter an *Endpoint name*, *GCP Project ID*, *Google Service Account Credentials*, and select **Create**.
```

Comment:
in the Aiven Client instructions there's more detail on these, especially on "Google Service Account Credentials". Maybe have those instruction centralized or have them also here?

Reply:
> in the Aiven Client instructions there's more detail on these, especially on "Google Service Account Credentials". Maybe have those instruction centralized or have them also here?

With the console's new side menu design, it's beneficial to have these instructions right there when users are doing tasks. This way, they don't have to keep going between different documents and screens. Having a link that takes them to another article for just four steps is unnecessary. The Google BigQUery integration doc in a common location (mostly Service integrations) is also still WIP. But once this is complete, I can update this article to include the link.